### PR TITLE
Fix Irene bug caused by Numpy upgrade

### PR DESCRIPTION
**Pull request by jacg**
We were using Numpy 1.14.5. 1.15.0 is available, but `test_irene_split_trigger` breaks if we use the new version of Numpy.

### CODE REVIEW

**Comment by jacg** [comment]
@jmbenlloch I've assigned this to you, as you seem to be the author of the test which is failing.

**Comment by jmbenlloch** [comment]
The problem here is that there are new warnings:

```
(Pdb) record[0].message
FutureWarning('Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.',)
(Pdb) record[1].message
FutureWarning('Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.',)
(Pdb) record[2].message
UserWarning('Event 2 has an unknown trigger type (5)',)
(Pdb) record[3].message
FutureWarning('Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.',)
```

The test checks irene raises a warning for unkown trigger (`record[2]`) (and only one). The rest of then come from:

```
  /home/jmbenlloch/miniconda/envs/IC-3.6-2018-07-28/lib/python3.6/site-packages/scipy/signal/signaltools.py:1344: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
    out = out_full[ind]
```

I don't know if this is going to be fixed soon by scipy...

**Comment by jacg** [comment]
I have modified the test, so that, rather than requiring exactly one warning with exactly that message, the test now requires at least one warning, exactly one of which has the required message. I think that is logically more correct: it's not this test's business whether any other warnings appear or not.

@jmbenlloch Please review.

**Comment by jacg** [comment]
Unfortunately the tests fail elsewhere about 50% of the time, probably because of spurious hypothesis problems, but that's a job for a separate PR.

**Comment by jmbenlloch** [comment]
> I have modified the test, so that, rather than requiring exactly one warning with exactly that message, the test now requires at least one warning, exactly one of which has the required message. I think that is logically more correct: it's not this test's business whether any other warnings appear or not.

I agree, that seems to be a good solution!

**Comment by jacg** [comment]
In which case, please approve the PR, if there's nothing else to consider. Of course it's a bit unfortunate that this one is on top of #493 which is still awaiting approval ...

**Review by jmbenlloch — APPROVED** [review]
You're right. This is approved, the bug has been successfully fixed.

**Comment by jacg** [comment]
@carmenromo I'm moving this to the *Merge* column, but **don't merge it until AFTER #493**

**Comment by jacg** [comment]
@carmenromo #493 has been merged, so please go ahead and merge this as soon as Travis gives its approval.

**Comment by carmenromo** [comment]
ok!

**Comment by carmenromo** [comment]
Done

**Comment by jacg** [comment]
@carmenromo Travis has given the go-ahead. Please merge ... ah I see you did it before I finished typing. Well done!

A minor comment about the merge commit text: The two parts of the PR description that appear in brackets aren't really parts of the description: they were warnings about not merging too soon. Now that it is ready to merge, these warnings don't apply, so they should not have been added to the PR description in the merge commit message, and they need to be edited out of the PR description on GitHub.

There's nothing that can be done about the merge commit now that it has been pushed (at least nothing that is worth the trouble it causes). But please do edit the PR description on GitHub, to remove the obsolete warnings.

**Comment by carmenromo** [comment]
Oh sorry... I edited it on GitHub

**Pull request merged**
